ml/ml.h

```diff
@@ -10,39 +10,30 @@ extern "C" {
 #include "daemon/common.h"
 #include "web/api/queries/rrdr.h"
 
-// This is a DBEngine function redeclared here so that we can free
-// the anomaly rate dimension, whenever its backing dimension is freed.
-void rrddim_free(RRDSET *rs, RRDDIM *rd);
```

Comment:
Probably because of this, we now have 

```
daemon/service.c:76:5: warning: implicit declaration of function 'rrddim_free'; did you mean 'rrdset_free'? [-Wimplicit-function-declaration]
   76 |     rrddim_free(st, rd);
```

probably need to be defined in `rrd.h`

Reply:
Should be fixed in a latest commit.